tests/Expected/Business/testJsonSerialize.json

```diff
@@ -0,0 +1,42 @@
+{
+  "days": {
+    "1": {
+      "dayOfWeek": 1,
+      "openingIntervals": [
+        {
+          "start": {
+            "hours": 9,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 13,
+            "minutes": 0,
+            "seconds": 0
+          }
+        },
+        {
+          "start": {
+            "hours": 14,
+            "minutes": 0,
+            "seconds": 0
+          },
+          "end": {
+            "hours": 17,
+            "minutes": 0,
+            "seconds": 0
+          }
+        }
+      ]
+    },
+    "5": {
+      "dayOfWeek": 5,
+      "openingIntervals": []
```

Comment:
How would code serialization look in JSON ? Also, does it make sense ? JSON should be used to send data across systems. If you read code in another system, you will not be able to always run it there, and you should not do that anyways. 

Imagine the JSON beeing used for an API. It does make sense to send out information regarding the `now` for every request. 

Or as i've written before, make `evaluateOpeningIntervals` public and let the developer specify the context